src/shared/components/blocks/catchphrase/Catchphrase.module.css

```diff
@@ -1,3 +1,36 @@
-.catchphrase {
-    background: palevioletred;
+@import "../../../styles/mixins/layout";
+@import "../../../styles/colors";
+@import "../../../styles/custom-medias";
+
+@mixin layout-main .container, .blackScreen;
+
+.container {
+    height: 90vh;
```

Comment:
the purpose of this container is to have 100vh, right?

Reply:
Not quite , according to the designs.